resources/lang/vi/community.php

```diff
@@ -22,108 +22,109 @@
+                'title' => 'Chỉnh Sửa',
+                'description' => 'Chỉnh sửa profile của bạn bằng cách thêm một user page đầy đủ có thể chỉnh sửa.',
             ],
 
             'beatmap_filters' => [
-                'title' => 'Beatmap Filters',
-                'description' => 'Filter beatmap searches by played and unplayed maps and rank achieved (if any).',
+                'title' => 'Lọc Beatmap',
+                'description' => 'Tìm kiếm beatmap được lọc theo đã chơi, chưa chơi và rank đạt được (nếu có).',
             ],
 
             'yellow_fellow' => [
-                'title' => 'Yellow Fellow',
-                'description' => 'Be recognised in-game with your new bright yellow chat username colour.',
+                'title' => 'Anh Bạn Màu Vàng',
+                'description' => 'Được nhận diện trong game bằng màu vàng sáng trên tên tài khoản của bạn trong chat.',
             ],
 
             'speedy_downloads' => [
-                'title' => 'Speedy Downloads',
-                'description' => 'More lenient download restrictions, especially when using osu!direct.',
+                'title' => 'Tải Nhanh',
+                'description' => 'Tải ít bị giới hạn hơn, đặc biệt là khi dùng osu!direct.',
             ],
 
             'change_username' => [
-                'title' => 'Change Username',
-                'description' => 'The ability to change your username without additional costs. (once max)',
+                'title' => 'Đổi Tên Tài Khoản',
+                'description' => 'Được đổi tên tài khoản miễn phí. (tối đa một lần)',
             ],
 
             'skinnables' => [
-                'title' => 'Skinnables',
-                'description' => 'Extra in-game skinnables, like the main menu background.',
+                'title' => 'Thêm Skin',
+                'description' => 'Thêm in-game skin, như là màn hình background ở menu chính.',
             ],
 
             'feature_votes' => [
-                'title' => 'Feature Votes',
-                'description' => 'Votes for feature requests. (2 per month)',
+                'title' => 'Bầu Chọn Tính Năng',
+                'description' => 'Bầu chọn cho các tính năng theo yêu cầu. (2 lần mỗi tháng)',
             ],
 
             'sort_options' => [
-                'title' => 'Sort Options',
-                'description' => 'NEW: The ability to view beatmap country / friend / mod-specific rankings in-game.',
+                'title' => 'Tùy Chọn Sắp Xếp',
+                'description' => 'MỚI: Cho phép xem xếp hạng theo quốc gia / bạn bè / theo-mod của beatmap trong game.',
             ],
 
             'feel_special' => [
-                'title' => 'Feel Special',
-                'description' => 'The warm and fuzzy feeling of doing your part to keep osu! running smoothly!',
+                'title' => 'Cảm Thấy Đặc Biệt',
+                'description' => 'Cảm giác ấm áp mờ khi chung tay giúp cho osu! chạy một cách trơn tru!',
```

Comment:
`Cảm giác ấm áp mờ` là sao :v

Reply:
This what happens when you translate warm and fuzzy too literally lol